src/test/ui/closures/closure-bounds-subtype.stderr

```diff
@@ -8,11 +8,10 @@ LL |     take_const_owned(f);
    |                      ^ `F` cannot be shared between threads safely
    |
    = help: the trait `std::marker::Sync` is not implemented for `F`
-help: consider further restricting this bound with `+ std::marker::Sync`
-  --> $DIR/closure-bounds-subtype.rs:11:30
+help: consider further restricting this bound
    |
-LL | fn give_owned<F>(f: F) where F: FnOnce() + Send {
-   |                              ^^^^^^^^^^^^^^^^^^
+LL | fn give_owned<F>(f: F) where F: FnOnce() + Send + std::marker::Sync {
```

Comment:
Could we check whether the path is in scope and strip the `std::marker::` bit? Happy to defer that to a follow up though!

Reply:
I would prefer to leave that for a different PR addressing referring to types with their locally known ident instead of adhoc for these suggestions only.

Reply:
Fair :)